### PR TITLE
Adding config for ICR davros cluster

### PR DESCRIPTION
**Pull request by adrlar**
Tested with sarek run successfully

### CODE REVIEW

File: conf/icr_davros.config [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * -------------------------------------------------
+ *  Nextflow nf-core config file for ICR davros HPC
+ * -------------------------------------------------
+ * Defines LSF process executor and singularity
+ * settings.
+ * 
+ */
+params {
+  config_profile_description = "Nextflow nf-core profile for ICR davros HPC"
+  config_profile_contact = "Adrian Larkeryd"
```
Comment (apeltzer) [inline conf/icr_davros.config:11]: Please add either github handle and/or email here so people can get in touch :-)

File: conf/icr_davros.config [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+singularity {
+  enabled = true
+  runOptions = "--bind /mnt:/mnt --bind /data:/data"
```
Comment (apeltzer) [inline conf/icr_davros.config:16]: Maybe you want `autoMounts = true ` did that not work for you?

**Comment by adrlar** [comment]
Awesome, thanks for checking - will add docs and look at the suggestions. Reason for naming is that IT is procuring a second, separate cluster at the moment so there will be two versions soon.

**Comment by apeltzer** [comment]
> Awesome, thanks for checking - will add docs and look at the suggestions. Reason for naming is that IT is procuring a second, separate cluster at the moment so there will be two versions soon.

That makes sense then - just wanted to check whether there is a reason for it :-)

**Comment by adrlar** [comment]
Changed up a couple of things according to the suggestions, and did more testing to make sure it works.

**Comment by adrlar** [comment]
I hadn't realised there was a yml file for saying the config was to be tested, have added that now. But I'm afraid of hitting large green buttons with scary text in them ... :)

**Comment by apeltzer** [comment]
Looks good , merged 👍

**Pull request merged**